distribution/docs/src/main/resources/_contents/_imaging/managing-imaging-contents.adoc

```diff
@@ -0,0 +1,44 @@
+
+The ${alliance-imaging} Application provides support for ingesting and searching for NITF products.
+
+=== Prerequisites for ${alliance-imaging}
+
+None.
+
+=== Installing ${alliance-imaging}
+
+This application is installed by default on a standard ${branding} installation.
```

Comment:
Change "This app..." to "The Alliance NSILI app..." to have the text independent of the heading.